tests/test-amp-style-sanitizer.php

```diff
@@ -679,6 +679,44 @@ public function test_get_validated_url_file_path( $source, $expected, $error_cod
 		}
 	}
 
+	/**
+	 * Get data URLs.
+	 *
+	 * @returns array data: URL data.
+	 */
+	public function get_data_urls() {
+		return array(
+			'url_with_spaces' => array(
+				'html { background-image:url(url with spaces.png); }',
+				'html{background-image:url("urlwithspaces.png")}',
+			),
```

Comment:
Should include `data:` URL as discussed. Include spaces as reported in support forums.